ci/build_setup.sh

```diff
@@ -101,5 +104,7 @@ function cleanup() {
   rm -f "${ENVOY_CI_DIR}"/bazel-*
   rm -rf "${ENVOY_CI_DIR}"/bazel
   rm -rf "${ENVOY_CI_DIR}"/tools
+  rm -f "${ENVOY_SRCDIR}"/ci/prebuilt/BUILD
+  touch "${ENVOY_SRCDIR}"/ci/prebuilt/BUILD
```

Comment:
Avoid leaving the tree in a dirty state once done, i.e. restore BUILD back to what it was.